build.rs

```diff
@@ -48,3 +83,67 @@ fn try_vcpkg() -> bool {
 fn try_vcpkg() -> bool {
     false
 }
+
+fn autogen_bindings() {
+    let lib_dir = if env::var("MYSQLCLIENT_LIB_DIR").is_ok() {
```

Comment:
Rather than hard coding system specifc path here I would like to use the same resolution logic as above.